Source/JavaScriptCore/jit/JITOpcodes.cpp

```diff
@@ -1668,7 +1668,8 @@ void JIT::emit_op_debug(const JSInstruction* currentInstruction)
     loadPtr(addressFor(CallFrameSlot::codeBlock), regT0);
     load32(Address(regT0, CodeBlock::offsetOfDebuggerRequests()), regT0);
     Jump noDebuggerRequests = branchTest32(Zero, regT0);
-    callOperation(operationDebug, TrustedImmPtr(&vm()), static_cast<int>(bytecode.m_debugHookType));
+    emitGetVirtualRegisterPayload(bytecode.m_data, regT1);
```

Comment:
Let's just get the data via `emitGetVirtualRegister`. And storing it to `jsRegT32`.